packages/11ty/.eleventy.js

```diff
@@ -170,11 +170,11 @@ module.exports = function(eleventyConfig) {
    * @property {Object} transformData - Additional global data for WebC transform
    * @property {Boolean} useTransform - Use WebC transform to process all HTML output
    */
-  eleventyConfig.addPlugin(pluginWebc, {
-    components: '_includes/components/**/*.webc',
-    transformData: {},
-    useTransform: false,
-  })
+  // eleventyConfig.addPlugin(pluginWebc, {
+  //   components: '_includes/components/**/*.webc',
+  //   transformData: {},
+  //   useTransform: false,
+  // })
```

Comment:
This is not using `pluginWebc` at the moment, `figure/image/sequence.js` uses the webc API directly